### ISSUE TITLE
Universität des Saarlandes not compatible

### ISSUE THREAD

**Issue by archi** [jakobwenzel/MensaGuthaben#31]
You list the UdS as compatible, but the App does not react to my card. According to NFC TagInfo (on the same device) it is a Mifare Classic 4k. Afaict only sectors 0 to 15 are used (16 to 39 use the default ffff keys).

Maybe the university issued/issues different cards (mine must be about 3 or 4 years old)?

**Comment by archi** [comment]
Newer cards seem to be DESfire; but I could not test them with your app.

**Comment by Natura2** [comment]
All newer cards for the Studirendenwerk Saarland are compatible. All Hochschulen and the Uni are using these.

Oldest I tested was from 2021, newest from end of 2025.

**Comment by archi** [comment]
Yeah, I think there is no point in supporting those decade-old cards 😆 Just look at the ancient Android version... Closing this issue then.
